Runtime/AvatarCreator/GenderSelectElement.cs

```diff
@@ -0,0 +1,45 @@
+using ReadyPlayerMe.Core;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace ReadyPlayerMe.AvatarCreator
+{
+    public class GenderSelectElement : MonoBehaviour
+    {
+        [Header("Buttons")]
+        [SerializeField, Tooltip("Button for selecting a male avatar")]
+        private Button maleButton;
+
+        [SerializeField, Tooltip("Button for selecting a female avatar")]
+        private Button femaleButton;
+
+        [Space(5)]
+        [Header("Events")]
+        public UnityEvent<OutfitGender> OnGenderSelected;
+
+        private void Awake()
+        {
+            InitializeButtons();
+        }
+
+        private void InitializeButtons()
+        {
+            if (maleButton != null)
+            {
+                maleButton.onClick.AddListener(() => OnGenderSelected?.Invoke(OutfitGender.Masculine));
```

Comment:
Small thing, but could you not use the same nullable operator that you use in onDestroy and drop the if statements?

```suggestion
                maleButton?.onClick.AddListener(() => OnGenderSelected?.Invoke(OutfitGender.Masculine));
```

Reply:
Yep the reason I didn't do it that way is using '?.' on objects deriving from UnityEngine.Object might not correctly reflect if it is null.
Unity implements/overrides the '==' operator to make up for this but it doesn't work on ?.. 
![image](https://github.com/readyplayerme/rpm-unity-sdk-core/assets/7085672/63886770-e4a9-4688-a85f-d4765bbbe34c)

Reply:
Ah, okay - annoying but makes sense ✅  - what do you think about changing the other one then so it's consistent? 🤔